aggregator/flush_mgr_options.go

```diff
@@ -35,7 +37,11 @@ const (
 )
 
 var (
-	defaultWorkerPoolSize = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
+	defaultWorkerPoolSize         = int(math.Max(float64(runtime.NumCPU()/8), 1.0))
```

Comment:
I did a global search and this is the only place in m3aggregator repo where `/8` is used. I picked this as the default since it has empirically worked well for us. Users in the OSS world of course can set different values in the config file to whatever value suits them the best.